linode/instance/schema_resource.go

```diff
@@ -29,6 +29,38 @@ You may need to switch to an explicit disk configuration.
 Take a look at the example here:
 https://www.terraform.io/docs/providers/linode/r/instance.html#linode-instance-with-explicit-configs-and-disks`
 
+func resourcePlacementGroup() *schema.Resource {
+	return &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"id": {
+				Type:        schema.TypeInt,
+				Required:    true,
+				Description: "The ID of the Placement Group to assign this Linode to.",
+			},
+			"compliant_only": {
+				Type:     schema.TypeBool,
+				Optional: true,
+			},
```

Comment:
I'm not sure if mixing computed & non-computed fields in a nested block is something we want to do here given the potential complications in framework 🤔

Reply:
I think it will trouble future framework work but I don't think there is a better way to implement it in SDKv2. So, let keep it?

Reply:
I think both are fine as we might have to migrate to nested attribute at the very very end.

Reply:
Sounds good to me! In that case I think it makes sense to keep things as-is 👍